cl/simple_pages/forms.py

```diff
@@ -6,6 +6,12 @@
 
 
 class ContactForm(forms.Form):
+    ISSUE_TYPE_CHOICES = [
+        ("removal", "Case Removal Request"),
+        ("bug", "RECAP Extension Bug"),
+        ("support", "Support"),
+    ]
+
```

Comment:
We should adopt Django's recommendation of using named constants for each value of the iterable.

```suggestion
    REMOVAL_REQUEST = "removal"
    RECAP_BUG = "recap"
    SUPPORT_REQUEST = "support"
    ISSUE_TYPE_CHOICES = [
        (REMOVAL_REQUEST, "Case Removal Request"),
        (RECAP_BUG, "RECAP Extension Bug"),
        (ISSUE_TYPE_CHOICES, "Support"),
    ]
   VALID_ISSUE_TYPES = [REMOVAL_REQUEST, RECAP_BUG, SUPPORT_REQUEST]
```

By following this suggestion, we can write cleaner and more expressive checks, such as:

```python
if form.issue_type == ContactForm.REMOVAL_REQUEST
```

and 

```python
if issue_type in ContactForm.VALID_ISSUE_TYPES:
```